packages/components/src/components/CancelButton/CancelButton.js

```diff
@@ -39,32 +40,60 @@ class CancelButton extends Component {
 
   render() {
     const { showDialog } = this.state;
-    const { type, name } = this.props;
+    const { intl, type, name } = this.props;
     const closeIcon = <StopFilled16 />;
     return (
       <>
         <button
           type="button"
-          title={`Stop ${type}`}
+          title={intl.formatMessage(
+            {
+              id: 'dashboard.cancelButton.tooltip',
+              defaultMessage: 'Stop {type}'
+            },
+            { type }
+          )}
           className="cancel-button"
           onClick={this.handleShow}
         >
           {closeIcon}
         </button>
         <Modal
           open={showDialog}
-          modalHeading={`Stop ${type} ${name}`}
-          primaryButtonText={`Stop ${type}`}
-          secondaryButtonText="Cancel"
+          modalHeading={intl.formatMessage(
+            {
+              id: 'dashboard.cancelButton.modalHeading',
+              defaultMessage: 'Stop {type} {name}'
+            },
+            { type, name }
+          )}
+          primaryButtonText={intl.formatMessage(
+            {
+              id: 'dashboard.cancelButton.primaryText',
+              defaultMessage: 'Stop {type}'
+            },
+            { type }
+          )}
+          secondaryButtonText={intl.formatMessage({
+            id: 'dashboard.cancelButton.secondaryText',
```

Comment:
should we have `dashboard.modal.cancelButton` or something generic like that and use it everywhere to ensure consistency?